agent/src/main/scala/za/co/absa/atum/agent/model/Measurement.scala

```diff
@@ -0,0 +1,130 @@
+package za.co.absa.atum.agent.model
+
+import org.apache.spark.sql.{Column, DataFrame, Dataset, Row}
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.types.{DecimalType, LongType, StringType}
+import za.co.absa.atum.agent.core.MeasurementProcessor
+import org.apache.spark.sql.functions._
+import za.co.absa.atum.agent.core.MeasurementProcessor.MeasurementFunction
+import za.co.absa.atum.agent.model.Measurement.RecordCount
```

Comment:
No need to import it, it's in this module after all. Maybe this is a refactoring thing, we all are doing these from time to time :D